WooCommercePOS/WooCommercePOS/Presentation/PlusButtonView.swift

```diff
@@ -0,0 +1,25 @@
+import SwiftUI
+
+struct PlusButtonView: View {
+    private var buttonTapped: (() -> Void)?
+
+    init(buttonTapped: (() -> Void)? = nil) {
+        self.buttonTapped = buttonTapped
+    }
+
+    var body: some View {
+        Button(action: {
+            buttonTapped?()
+        }) {
+            Image(systemName: "plus")
+                .font(.system(size: 24, weight: .bold))
+                .foregroundColor(.white)
+                .frame(width: 50, height: 50)
+                .background(Circle().fill(Color.clear))
```

Comment:
just curious, is this Circle background mean to have a circle around the plus like in design? The current UI doesn't seem to have the circle. Alternatively, we can use `plus.circle` symbol for the image `Image(systemName: "plus.circle")`.

Reply:
That was the idea yes, for some reason I used .clear and forgot about it 😅 . I've deleted the PlusButtonView on https://github.com/woocommerce/woocommerce-ios/pull/12720/commits/5afe603b7633faa5e6d7199bd4556d11168f42cf in any case, since we apply now the "plus button" view modifier.

Reply:
> we can use plus.circle symbol for the image Image(systemName: "plus.circle").

Good tip, updated the image on https://github.com/woocommerce/woocommerce-ios/pull/12720/commits/46bc77c605b3e0bc99560bb89526b76b08c6aeb7